src/librustdoc/json/mod.rs

```diff
@@ -244,7 +243,7 @@ impl<'tcx> FormatRenderer<'tcx> for JsonRenderer<'tcx> {
                     )
                 })
                 .collect(),
-            format_version: 4,
+            format_version: 5,
```

Comment:
I dont think we need to change this, as their are no changes to the output, right?

Reply:
Right, I'll change it back. For some reason I thought I'd changed something, but you're right